i18n/README-ZH.md

```diff
@@ -101,28 +105,31 @@ Flame的官方网站(其中包含文档), 请[点击此处](https://flame-engine
 
 ## 贡献
 
-__注意__: 我们目前在着手于发布Flame的第一个稳定版本, 所以, 除Bug修复外, `0.x`的版本更新将全部冻结. 如果你想贡献该版本, 请确保你的提交是修复Bug. 若想贡献稳定版本, 请务必将你的PR指向`v1.0.0`分支, 并且一定要将你的贡献讲出来, 并且可在 [Discord](https://discord.gg/pxrBmy4) 上访问该分支.
+**注意**：我们目前在着手于发布 Flame 的第一个稳定版本，所以，除了严重的 BUG 修复以外，`0.x` 的版本更新将全部冻结。
+如果你想贡献该版本，请确保你的提交是在修复 BUG。
+若想贡献至稳定版本，请务必将你的 PR 指向 `main` 分支，并且与团队沟通你的贡献，
+可以是 issue 方式，或在 [Discord](https://discord.gg/pxrBmy4) 上沟通。
 
-我们感激你以任何形式帮助我们! 评论, 建议, 提issues, 以及PR(Pull Request).
+我们感激你以任何形式帮助我们！评论、建议、提 issues、或是 PR (Pull Request)。
 
-如果你找到了bug或者有使Flame更好的建议, 创建一个issue. 我们将会尽快处理.
+如果你找到了 bug 或对 Flame 有更好的建议，请创建一个 issue，我们将会尽快处理。
 
-你想通过PR做贡献吗? PR总是受欢迎的, 只要确保你的分支是`develop`, 并遵循[PR模板](.github/pull_request_template.md)即可.
+你想通过 PR 进行贡献吗？我们欢迎 PR，只要确保你的分支是 `develop`，并遵循 [PR 模板](.github/pull_request_template.md) 即可。
 
 ## 立即开始
 
-简体中文教程 [Alekhin](https://github.com/japalekhin) Tutorial Series (Simplified Chinese) by [HarrisonQI](https://github.com/HarrisonQi) 
```

Comment:
We should keep those articles. There's so many Chinese users because those translated articles.

Reply:
Could you provide a repo link instead of a user profile link?

Reply:
Ok, you may remove. We'll update these articles to flame-cn.com